loc/views.py

```diff
@@ -124,6 +124,7 @@ def get_letter_context(user: JustfixUser) -> Dict[str, Any]:
         'landlord_details': get_landlord_details(user),
         'onboarding_info': get_onboarding_info(user),
         'issues': get_issues(user),
+        'has_heat_issues': any(s in str(get_issues(user)).upper() for s in ('HEAT', 'HOT WATER')),
```

Comment:
If this ends up really being a `has_issues_that_qualify_for_an_emergency_hp` (well less verbose probably), it will be useful to pull in some of the code from #1063 to make everything more DRY.

Reply:
See https://github.com/JustFixNYC/tenants2/pull/1050#discussion_r399438259